flag/service/helm/helm.go

```diff
@@ -6,8 +6,15 @@ import (
 )
 
 type Helm struct {
-	HTTP            http.HTTP
-	Kubernetes      kubernetes.Kubernetes
-	MaxRollback     string
+	HTTP        http.HTTP
+	Kubernetes  kubernetes.Kubernetes
+	MaxRollback string
+
+	// SplitClient determines usage of additional pubHelmClient impersonating
+	// `default:automation` Service Account for App CRs created outside the
+	// `giantswarm` namespace. When `false` Chart Operator runs under full
+	// cluster admin permissions no matter the App CR namespace.
+	SplitClient string
```

Comment:
It's defined as `bool` [here](https://github.com/giantswarm/chart-operator/pull/875/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R112). Which is it?